pkg/sfu/buffer/rtpstats.go

```diff
@@ -212,8 +212,8 @@ func (r *RTPStats) Update(rtph *rtp.Header, payloadSize int, paddingSize int, pa
 		}
 		// LK-DEBUG-REMOVE START
 		if diff > 100 {
-			logger.Debugw(
-				"DEBUG huge difference in sequence number",
+			logger.Errorw(
+				"DEBUG huge difference in sequence number", nil,
```

Comment:
"DEBUG huge ..." change to "ERROR huge ..."?

Reply:
It is just temporary @cnderrauber . I am just going to search by `huge difference`. Just wanted a back trace to see where it is coming from.  Will remove it entirely once we understand this better.